templates/index.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'layout.html' %}
+
+{% block title %}Введите данные{% endblock %}
+
+{% block content %}
+    <h2>Введите данные</h2>
+    <form action="/welcome" method="post">
```

Comment:
Тогда атрибут `action` больше не понадобиться
```suggestion
    <form method="post">
```